software/nginx/en.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "nginx"
+lastmod = "2020-04-03T23:40:51+02:00"
++++
+# nginx
+
+Solus provides the nginx web server and proxy to users and developers who want to test their web applications before setting them to production.
+Remember that Solus is not a server operating system: any usage outside the local testing is discouraged.
```

Comment:
When I mentioned the \, I kinda figured you'd put a proper new line in rather than just a line break. We don't really need to do those :P Let's put a newline between 7 and 8.